package.json

```diff
@@ -69,6 +69,7 @@
   },
   "devDependencies": {
     "@balena/lint": "^8.2.8",
+    "@balena/pinejs-webresource-s3": "1.0.1",
```

Comment:
Any reason for a pinned version @otaviojacobi ?

Reply:
Nope, I just didn't realize NPM added it pinned by default. thanks